darts/models/theta.py

```diff
@@ -115,14 +120,241 @@ def predict(self, n: int) -> 'TimeSeries':
 
             replicated_seasonality = np.tile(self.seasonality.pd_series()[-self.season_period:],
                                              math.ceil(n / self.season_period))[:n]
-            if self.mode in ['multiplicative', 'mul']:
+            if self.season_mode is Season.MULTIPLICATIVE:
                 forecast *= replicated_seasonality
-            elif self.mode in ['additive', 'add']:
+            elif self.season_mode is Season.ADDITIVE:
                 forecast += replicated_seasonality
-            else:
-                raise ValueError("mode cannot be {}".format(self.mode))
 
         return self._build_forecast_series(forecast)
 
     def __str__(self):
         return 'Theta({})'.format(self.theta)
+
+
+class FourTheta(UnivariateForecastingModel):
+    def __init__(self,
+                 theta: int = 2,
+                 seasonality_period: Optional[int] = None,
+                 season_mode: Season = Season.MULTIPLICATIVE,
+                 model_mode: Model = Model.ADDITIVE,
+                 trend_mode: Trend = Trend.LINEAR,
+                 normalization: bool = True):
+        """
+        An implementation of the 4Theta method with configurable `theta` parameter.
+
+        See M4 competition `solution <https://github.com/Mcompetitions/M4-methods/blob/master/4Theta%20method.R>`_.
+
+        The training time series is de-seasonalized according to `seasonality_period`,
+        or an inferred seasonality period.
+
+        When called with `theta = 2 - X`, `model_mode = Model.ADDITIVE` and `trend_mode = Trend.LINEAR`,
+        this model is equivalent to calling `Theta(theta=X)`.
+
+        Parameters
+        ----------
+        theta
+            Value of the theta parameter. Defaults to 2.
+            If theta = 1, then the fourtheta method restricts to a simple exponential smoothing (SES).
+            If theta = 0, then the fourtheta method restricts to a simple `trend_mode` regression.
+        seasonality_period
+            User-defined seasonality period. If not set, will be tentatively inferred from the training series upon
+            calling `fit()`.
+        model_mode
+            Type of model combining the Theta lines. Must be a Model Enum member. Defaults to `Model.ADDITIVE`.
```

Comment:
Same comment as above, let's link to a place where users can see all possible values for the Enums